docs(GcpRedisInstance): add redisTier details (#787)
kyma-project/cloud-manager: docs/user/resources/04-60-gcp-redis-instance.md

If the default IpRange does not exist, it is automatically created.
Manually create a non-default IpRange with specified Classless Inter-Domain Routing (CIDR) and use it only in advanced cases of network topology when you want to control the network segments to avoid range conflicts with other networks.

When creating GcpRedisInstance, only one field is mandatory: `redisTier`.
It specifies service tier (*Standard* or *Premium*), and capacity tier.
More details regarding tiers can be found in the chapter below.

Optionally, you can specify the `redisVersion`, `authEnabled`, `redisConfigs`, and `maintenancePolicy` fields.

As in-transit encryption is always enabled, communication with the Redis instance requires a certificate. The certificate can be found in the Secret on the `.data.CaCert.pem` path.

# Redis Tiers

## Standard Tier

> In the *Standard* service tier, instance does not have a replica, and thus it cannot be considered highly available.

Reviewer: ```suggestion
In the **Standard** service tier, the instance does not have a replica. Thus, it cannot be considered highly available.
```

| RedisTier | Capacity (GiB) | Network (minimum Gbps) |
| --------- | -------------- | ---------------------- |
| S1        | 1              | 10                     |
| S2        | 3              | 10                     |
| S3        | 6              | 10                     |
| S4        | 12             | 10                     |
| S5        | 24             | 10                     |
| S6        | 48             | 16                     |
| S7        | 101            | 16                     |
| S8        | 200            | 16                     |

## Premium tier

> In the *Premium* service tier, instance comes with a read replica, and automatic failover enabled. This instance can be considered highly available.

Reviewer: ```suggestion
In the **Premium** service tier, the instance comes with a read replica and automatic failover enabled. Thus, it can be considered highly available.
```

| --------------------------------------------------| ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| **ipRange**                                       | object | Optional. IpRange reference. If omitted, the default IpRange is used. If the default IpRange does not exist, it will be created.                                                                            |
| **ipRange.name**                                  | string | Required. Name of the existing IpRange to use.                                                                                                                                                              |

> | **redisTier**                                     | string | Required. The redis tier of the instance. Supported values are `S1`, `S2`, `S3`, `S4`, `S5`, `S6`, `S7`, `S8` for the *Standard* offering, and `P1`, `P2`, `P3`, `P4`, `P5`, `P6` for the *Premium* offering. |

Reviewer: ```suggestion
| **redisTier**                                     | string | Required. The Redis tier of the instance. Supported values are `S1`, `S2`, `S3`, `S4`, `S5`, `S6`, `S7`, `S8` for the **Standard** offering, and `P1`, `P2`, `P3`, `P4`, `P5`, `P6` for the **Premium** offering. |
```
